dotcom-rendering/src/web/experiments/tests/sign-in-gate-copy-test-variants.ts

```diff
@@ -0,0 +1,32 @@
+import { ABTest } from '@guardian/ab-core';
+
+export const signInGateCopyTestVariants: ABTest = {
```

Comment:
I would be consistent with the name and use either signInGateCopyTestVariants or signInGateCopyTestJan2023

Reply:
Sure, I've updated this